### DISCUSSION TITLE
Android: creating new issue auto-populates with previously created issue

### DISCUSSION THREAD

**Discussion by charlie-wiser** [community/community#12671] [Mobile]
Hey there - this appears to be a bug on the Android app. Whenever I create a new issue, it auto-populates with the issue that I previously created. Thanks.

**Comment by aconsuegra** [comment]
Hey @charlie-wiser, thanks for the bug report. I have been able to reproduce it, so I'm going to create an issue so our engineers can work on a fix as soon as possible .

**Reply by aconsuegra** [reply]
@charlie-wiser following up here. The bug has been fixed and will be included in our upcoming release. Thanks again for reporting it!
